### DISCUSSION TITLE
Nickelodeon India

### DISCUSSION THREAD

**Discussion by Niyasmkkv** [iptv-org/community#694] [Q&A]
**Channel Name:** Nick Malayalam
**Channel ID:** Nickelodeon.in
**Country:** IN
**Language:** Malayalam
**Website:** xxx
**Stream URL (optional):** https://tvstream.space/b0807355-aeba-4dbd-a569-02c73e2eb808
**Notes (optional):** xxx

**Comment by freearhey** [comment]
Are you looking for this channel: https://en.wikipedia.org/wiki/Nickelodeon_(Indian_TV_channel)?

**Comment by Niyasmkkv** [comment]
Yup. That channel's malayalam audio feed.

**Comment by BellezaEmporium** [comment]
Geolocked Indian IP, got to it via a proxy. 
User-Agent needed = Mozilla/5.0 (iPhone; CPU iPhone OS 10_3_1 like Mac OS X) AppleWebKit/603.1.30 (KHTML, like Gecko) Version/10.0 JioTV Mobile/14E304 Safari/602.1

![image](https://user-images.githubusercontent.com/30985701/130361739-c03e03ab-ab18-4521-8a7e-34c0a7c38cf3.png)

Hmm, gonna need to investigate that.

Nick Jr :
https://mumsite.cdnsrv.jio.com/jiotv.live.cdn.jio.com/Nick_Junior/Nick_Junior_600.m3u8?jct=LrgwZm7NGzQYbOWCSflvaw&pxe=1581172404&st=2rABpyTLSDzkoYSlKcWJhg
Nick Tamil : 
http://mumsite.cdnsrv.jio.com/jiotv.live.cdn.jio.com/Nick_Tamil/Nick_Tamil_600.m3u8?jct=LrgwZm7NGzQYbOWCSflvaw&pxe=1581172404&st=2rABpyTLSDzkoYSlKcWJhg
Nick Telugu : 
http://mumsite.cdnsrv.jio.com/jiotv.live.cdn.jio.com/Nick_Telugu/Nick_Telugu_600.m3u8?jct=LrgwZm7NGzQYbOWCSflvaw&pxe=1581172404&st=2rABpyTLSDzkoYSlKcWJhg
Nick Hindi :
http://mumsite.cdnsrv.jio.com/jiotvstb.live.cdn.jio.com/Nick_Hindi/Nick_Hindi_600.m3u8?jct=LrgwZm7NGzQYbOWCSflvaw&pxe=1581172404&st=2rABpyTLSDzkoYSlKcWJhg

**Comment by rsk1987** [comment]
> Geolocked Indian IP, got to it via a proxy. User-Agent needed = Mozilla/5.0 (iPhone; CPU iPhone OS 10_3_1 like Mac OS X) AppleWebKit/603.1.30 (KHTML, like Gecko) Version/10.0 JioTV Mobile/14E304 Safari/602.1
> 
> ![image](https://user-images.githubusercontent.com/30985701/130361739-c03e03ab-ab18-4521-8a7e-34c0a7c38cf3.png)
> 
> Hmm, gonna need to investigate that.
> 
> Nick Jr : https://mumsite.cdnsrv.jio.com/jiotv.live.cdn.jio.com/Nick_Junior/Nick_Junior_600.m3u8?jct=LrgwZm7NGzQYbOWCSflvaw&pxe=1581172404&st=2rABpyTLSDzkoYSlKcWJhg Nick Tamil : http://mumsite.cdnsrv.jio.com/jiotv.live.cdn.jio.com/Nick_Tamil/Nick_Tamil_600.m3u8?jct=LrgwZm7NGzQYbOWCSflvaw&pxe=1581172404&st=2rABpyTLSDzkoYSlKcWJhg Nick Telugu : http://mumsite.cdnsrv.jio.com/jiotv.live.cdn.jio.com/Nick_Telugu/Nick_Telugu_600.m3u8?jct=LrgwZm7NGzQYbOWCSflvaw&pxe=1581172404&st=2rABpyTLSDzkoYSlKcWJhg Nick Hindi : http://mumsite.cdnsrv.jio.com/jiotvstb.live.cdn.jio.com/Nick_Hindi/Nick_Hindi_600.m3u8?jct=LrgwZm7NGzQYbOWCSflvaw&pxe=1581172404&st=2rABpyTLSDzkoYSlKcWJhg

Can you provide current link ? because link not working that you provided.

**Comment by freearhey** [comment]
Closed in favor of https://github.com/iptv-org/iptv/issues/19719

**Comment by MahirByte** [comment]
HEY, How About the Nick Bengali Language

**Comment by MahirByte** [comment]
How About Nickelodeon with Bengali Language
